R/chk-all.R

```diff
@@ -32,6 +32,30 @@ chk_all <- function(x, chk_fun, ..., x_name = NULL) {
   do.call("lapply", args)
   invisible()
 }
+#' @describeIn chk_all Check All
+#'
+#' @examples
+#' # check_all
+#' check_all(TRUE, chk_lgl)
+#' # FIXME try(check_all(1, chk_lgl))
+#' check_all(c(TRUE, NA), chk_lgl)
+#' @export
+check_all <- function(x, chk_fun, ..., x_name = NULL) {
+  if (is.null(x)) {
+    if (is.null(x_name)) x_name <- deparse_backtick_chk(substitute(x))
+    return(chk_fun(x, ..., x_name = x_name))
```

Comment:
```suggestion
    chk_fun(x, ..., x_name = x_name)
    return(invisible(x))
```

Reply:
@nadinehussein - The changes look great - this is the only suggested change I have.